selfdrive/modeld/parse_model_outputs.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+from typing import Dict
+from openpilot.selfdrive.modeld.constants import (
```

Comment:
I see this is imported in a few places. instead of these long imports, it's best to generally namespace with the module or the class, e.g. `ModelConstants.PLAN_MHP_N`

Reply:
added a class for ModelConstants